home/ivp-planning/getting-started/importing-pdfs.md

```diff
@@ -2,30 +2,28 @@
 
 In iVP it is possible to import PDFs that will act as an overlay for the [floor plan](../user-interface/the-floor-plan.md). This can be useful for illustration or when it is necessary to transform a 2D (factory) plan into a 3D scene.
 
+## Importing PDFs step by step:
 
-{% embed url="https://youtu.be/MuLt94b64O8?t=203" %}
-How to import and export PDFs (german)
-{% endembed %}
+To import a PDF, click on **File -> Import**. A new window will open that allows you to navigate to the folder containing the PDF you want to import.
 
-## Importing PDFs step by step:
+![](../../../.gitbook/assets/pdf_import_pdf_menu_entry.jpg)
+
+As soon as you have found the file, simply double-click on it or select it and click **Open**.
+
+![](../../../.gitbook/assets/pdf_import_pdf_file_dialogue.jpg)
```

Comment:
The import works the same for all file types and selecting a file in a file browser probably does not need to be explained. I just explaining where the import button is should be enough.